src/screens/settings/Settings.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  TouchableOpacity,
+  Dimensions,
+} from 'react-native';
+import {useTranslation} from 'react-i18next';
+
+const {width} = Dimensions.get('window');
+
+const LANGS = [
```

Comment:
these data we can put in the Data.ts file inside services folder

Reply:
resolved